### DISCUSSION TITLE
Video about django-components

### DISCUSSION THREAD

**Discussion by EmilStenstrom** [django-components/django-components#550] [Show and tell]
https://www.youtube.com/watch?v=Pnp9IIxOXPU

Very interesting to see someone else describe how django-components works, and how to set it up.

**Comment by EmilStenstrom** [comment]
Quick thought: Why do we require changes to APP_DIRS and adding the cached template loader? Could things work without them?

**Reply by EmilStenstrom** [reply]
Someone annoyed by this: https://www.youtube.com/watch?v=Pnp9IIxOXPU&lc=UgxF1efjVCn2ropPuWR4AaABAg (scroll down to the first comment)

**Reply by JuroOravec** [reply]
Also not sure why `APP_DIRS` needs to be off. I made a ticket out of this https://github.com/EmilStenstrom/django-components/issues/558
